miso-web/src/main/webapp/scripts/list.js

```diff
@@ -99,6 +99,10 @@ ListUtils = (function() {
     "box": {
       term: "box:NAME",
       help: "Checks if an item is located in a particular box.  The name can either be the partial name or partial alias of the box."
+    },
+    "kitname": {
+      term: "kitname:NAME",
+      help: "Checks if an item contains a kit of the specified name."
```

Comment:
Probably worth having a separate message for containers to specify that it's checking clustering and multiplexing kits; otherwise, users might expect it to list containers that contain a library using a particular library kit. Or maybe just changing the word 'contains' to 'uses' makes it clear enough